### PR TITLE
Updated detector baseline and add material scans

### PR DESCRIPTION
**Pull request by jlingema**
- Based on: Event display discussed in PR #123.
- Updates master dimensions according to detector baseline discussed in the FCC-hh detector meeting on September 28th: https://indico.cern.ch/event/557688/
- Add a script that allows to scan distribution of interaction & radiation lengths as a function of eta

### CODE REVIEW

File: Detector/DetCommon/src/SimpleLayeredCylinder_geo.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  DD4hep::Geometry::Tube envelope(dimensions.rmin(), dimensions.rmax(), dimensions.dz());
+  DD4hep::Geometry::Volume envVolume(name, envelope, lcdd.material(dimensions.materialStr()));
+
+  // Create layer cylinders with their respective material, etc
+  auto layers = xmlElement.children("layer");
+  auto numLayers = xmlElement.numChildren("layer", true);
+  // Joschka: Although this is awkward, it was the only way to loop through children I could find
+  DD4hep::XML::Handle_t layer(layers.reset());
+  for(unsigned layerIdx = 0; layerIdx < numLayers; ++layerIdx) {
+    std::cout << layerIdx << std::endl;
```
Comment (zaborowska) [inline Detector/DetCommon/src/SimpleLayeredCylinder_geo.cpp]: I'm guessing a forgotten debug cout?

File: Detector/DetComponents/src/MaterialScan.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#include "DetInterface/IGeoSvc.h"
+
+#include "GaudiKernel/Service.h"
+
```
Comment (zaborowska) [inline Detector/DetComponents/src/MaterialScan.h]: no doxy comment here?

File: Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml [hunk 3]
```diff
@@ -5,121 +5,120 @@
 
   <info name="FCCDectDimensions"
     title="master file with includes and world dimension"
-    author="C. Helsens and J. Lingemann"
+    author="C. Helsens, J. Lingemann"
     url="no"
     status="development"
     version="$Id:??$">
```
Comment (zaborowska) [inline Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml]: are we going to start using this field at some point? Maybe now is a good moment?

File: Visualization/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+set(CMAKE_MODULE_PATH  ${CMAKE_MODULE_PATH}  ${DD4hep_ROOT}/cmake )
+find_package(DD4hep COMPONENTS DDCore DDG4 DDEve REQUIRED)
+
+include(DD4hep)
+
+find_package(FCCEDM)
+find_package(PODIO)
+
+
+gaudi_add_library(Visualization   NO_PUBLIC_HEADERS          src/*.cpp
```
Comment (zaborowska) [inline Visualization/CMakeLists.txt]: for clarity it'd be nice to break a line here

File: Visualization/src/FCCEventHandler.cpp [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+  else
+    return NO_COLLECTION;
+}
+
+template<typename T>
+void clusterEveConversion(T& clustercoll, DDEveHitActor& actor) {
+  int collsize = clustercoll->size();
+  actor.setSize(collsize);
+  for (const auto& cluster : *clustercoll) {
+    fcc::BareCluster core = cluster.Core();
```
Comment (zaborowska) [inline Visualization/src/FCCEventHandler.cpp]: Actually, after the recent changes in the EDM I believe we'd like it to use the positioned hit, not the cluster

**Comment by jlingema** [comment]
Any opinion on where the plotting script should go? I've put it in the examples for now, but I guess fcc-physics would be a better place?

**Pull request merged**